app/src/main/java/com/example/grutor/Utility/StudentMatcher.java

```diff
@@ -68,9 +68,11 @@ public void secondaryMatches(int rank) {
 
     public void getMyMatches() throws ParseException {
         List<ParseUser> studentTutors = new ArrayList<>();
+        Boolean matched = false;
         for (int i = 1; i < 3; i++) {
-            if (studentTutors.size() > 0) {
-                break;
+            if (studentTutors.size() > 0 && i >=1) {
+                matches.addAll(studentTutors);
+                matched = true;
             }
             if (i == 1) {
                 priorityMatches(i);
```

Comment:
above code is a little confused. it is hard to understand.
1. why need a loop?
2. why the loop start from 1, then end with < 3?
3. since only i==1 call priorityMatches(), why do you need priorityMatches() take i as parameter, what does it mean?